clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/screenshare/services/ScreenshareConnection.as

```diff
@@ -0,0 +1,161 @@
+package org.bigbluebutton.lib.screenshare.services
+{
+
+	import flash.net.NetConnection;
+	import flash.net.Responder;
+	
+	import org.bigbluebutton.lib.common.services.DefaultConnectionCallback;
+	import org.bigbluebutton.lib.common.services.IBaseConnection;
+	import org.bigbluebutton.lib.main.models.IConferenceParameters;
+	import org.osflash.signals.ISignal;
+	import org.osflash.signals.Signal;
+
+	public class ScreenshareConnection extends DefaultConnectionCallback implements IScreenshareConnection

```

Comment:
Duplicate class. This one can be removed.

Reply:
Connection dedicated to red5 screenshare app. Followed how we created connection classes for red5 video and red5 voice apps.

Reply:
Deleted.